packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -24,90 +24,48 @@ This guide explains how to use Okta as the user store for your server-side web a
 1. Click **Save** to finish creating the Okta app integration.
-1. On the **General** tab, the **Client Credentials** section shows the client ID and client secret values for your app integration.
-1. Copy the **Client ID** and **Client secret** values using the **Copy to Clipboard** button beside each text field.
-You need to copy some values into your application later, so leave your Admin Console open.
-1. In the **General Settings** section, click **Edit** and scroll down to **Login**. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow. When Okta redirects to this endpoint (for example, `https://example.com/login`), the client is triggered to send an authorize request. This URI is also used when users reset their passwords while signing in to the app. Okta redirects the user back to this URI after the password is reset so that the user can continue to sign in.
 
-### Enable refresh token rotation
-
-You can choose to [get a refresh token](/docs/guides/refresh-tokens/main/#get-a-refresh-token) along with the access token and/or ID token.
-
-The default refresh token behavior is **Use persistent token** for web apps.
-
-To enable refresh token rotation in your app integration, do the following:
-
-1. Open the web app integration that you just created and select the **General** tab.
-1. Scroll to the **General Settings** panel, and click **Edit**.
-1. In the **Allowed grant types**, select **Refresh Token**.
-1. In the **Refresh Token** section, select **Rotate token after every use**.
-
-> **Note:** The default number of seconds for the **Grace period for token rotation** is set to 30 seconds. You can [change the value](/docs/guides/refresh-tokens/main/#enable-refresh-token-rotation) to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there is no grace period.
-
-### Enable Trusted Origins
-
-To reduce possible attack vectors, you need to explicitly define the Trusted Origins that can access the Okta API for your app integration. Cross-Origin Resource Sharing (CORS) allows JavaScript hosted on your website to make a request using `XMLHttpRequest` to the Okta API with the Okta session cookie. For instructions on setting trusted origins, see [Grant cross-origin access to websites](/docs/guides/enable-cors/main/#grant-cross-origin-access-to-websites).
-
->**Note:** You should only grant access to specific origins (websites) that you control and trust to access the Okta API.
+-->
 
 ## Add and configure packages
 
-Next you need to add an Okta SDK to your application.
+Next you need to add the Okta Spring Boot Starter to your application.
```

Comment:
It'd be nice if we didn't have to use "an Okta SDK" as a generic term and could specify the exact name here.